test/simplifyExpression/simplify.test.js

```diff
@@ -91,6 +91,18 @@ describe('distribution', function () {
     ['(x-2)(x-4)', 'x^2 - 6x + 8'],
     ['- x*y^4 (6x * y^2 + 5x*y - 3x)',
       '-6x^2 * y^6 - 5x^2 * y^5 + 3x^2 * y^4'],
+    ['(nthRoot(x, 2))^2', 'x'],
+    // TODO: fix nthRoot to evaluate nthRoot(x^3, 2)
+    // ['(nthRoot(x, 2))^3', 'nthRoot(x ^ 3, 2)'],
+    // ['3 * nthRoot(x, 2) * (nthRoot(x, 2))^2', '3 * nthRoot(x ^ 3, 2)'],
+    // TODO: fix when base has multiplication
+    // ['(nthRoot(x, 2) * nthRoot(x, 2))^2', 'x^2'],
+    // ['(nthRoot(x, 2) * nthRoot(x, 3))^2', '(nthRoot(x, 2) * nthRoot(x, 3))^2'],
+    ['(nthRoot(x, 2))^4', 'x^2'],
+    ['3 * (nthRoot(x, 2))^2', '3x'],
+    ['(nthRoot(x, 2))^6 * (nthRoot(x, 3))^3', 'x^4'],
+    ['(2x + 3)^2','4x^2 + 12x + 9'],
+    ['(x + 3 + 4)^2', 'x^2 + 14x + 49'],
```

Comment:
Wait, well shouldn't we have substeps for `(x-2)^2`? Is this a unary minus issue? If so I think we should fix it.